api/apps/api/src/modules/planning-units/planning-units-completion.service.ts

```diff
@@ -74,6 +74,6 @@ export class PlanningUnitsCompletionService
 
   private async completed(event: EventData<PlanningUnitsJob>) {
     const jobData = await event.data;
-    await this.eventBus.publish(new PlanningUnitSet(jobData.projectId));
+    this.eventBus.publish(new PlanningUnitSet(jobData.projectId));
```

Comment:
why the removed `await`s here and in a couple more occurrences below, including in a test?

Reply:
`publish` method of `EventBus` is synchronous [link](https://github.com/nestjs/cqrs/blob/744ef1b409168f6f1fa146b670860a8d9ae89a80/src/event-bus.ts#L55)

Reply:
We realized that `.publish()` is sync